site/reference/pkg/init/README.md

```diff
@@ -9,25 +9,37 @@ description: >
     Initialize an empty package
 -->
 
-{{< asciinema key="pkg-init" rows="10" preload="1" >}}
+`init` initializes an existing empty directory as an empty kpt package.
 
-Init initializes an existing empty directory as an empty kpt package.
+### Synopsis
+<!--mdtogo:Long-->
+```
+kpt pkg init [DIR] [flags]
+```
+
+#### Args
+
+```
+DIR:
+  init fails if DIR does not already exist. Defaults to the current working directory.
+```
 
-**init is optional**: Any directory containing Kubernetes Resource
-Configuration may be treated as remote package without the existence of
-additional packaging metadata.
+#### Flags
 
-* Resource Configuration may be placed anywhere under DIR as *.yaml files.
-* DIR may contain additional non-Resource Configuration files.
-* DIR must be pushed to a git repo or repo subdirectory.
+```
+--description
```

Comment:
How are these flags decided ? There are other properties for of Kptfile which can be flags. e.g. Email

Reply:
That is a good question. This documents the flags that are currently expose on the `kpt pkg init` command. We probably should look into whether these are actually the right set of flags for the command, but lets handle that in a separate PR.